mis_builder_demo/models/mis_committed_purchase.sql

```diff
@@ -74,12 +74,12 @@ CREATE OR REPLACE VIEW mis_committed_purchase AS (
           WHEN (ail.price_subtotal / COALESCE(cur.rate, 1.0))::decimal(16,2)  < 0 THEN (ail.price_subtotal / COALESCE(cur.rate, 1.0))::decimal(16,2)
           ELSE 0.0
         END AS credit
-        FROM account_invoice_line ail
-            LEFT JOIN account_invoice ai ON ai.id = ail.invoice_id
+        FROM account_move_line ail
+            LEFT JOIN account_move ai ON ai.id = ail.move_id
```

Comment:
```suggestion
            LEFT JOIN account_move ai ON (ai.id = ail.move_id and not ail.exclude_from_invoice_tab)
```
the same below in line 106

Reply:
Ah, yes, this horror. Done